### PR TITLE
chore(ticdc): add OWNERS file in cdc api folder

### LINKED ISSUE #10819: Setup OWNERS on cdc/api/v2 folder for CDC API change approval
### Before asking a question, make sure you have

- [X] Searched existing Stack Overflow questions
- [X] Googled your question
- [X] Searched open and closed [GitHub issues](https://github.com/pingcap/tiflow/issues?q=is%3Aissue)
- [X] Read the [documentation](https://docs.pingcap.com/tidb/stable)

### What is your question?

Like https://github.com/pingcap/tidb/issues/45183

Scope: `cdc/api/v2` go source files
Approvers: @flowbehappy @sdojjy

### PR DESCRIPTION
**Pull request by wuhuizuo**
Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### What problem does this PR solve?

Issue Number: close #10819

#### Tests 

 - No code

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by wuhuizuo** [comment]
/cc @flowbehappy @okJiang 
PTAL

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Sort the member alphabetically.
+aliases:
+  sig-approvers-cdc-api:
+    - flowbehappy
+    - okJiang
```
Comment (sdojjy) [inline OWNERS_ALIASES:5]:
```suggestion
    - sdojjy
```

**Comment by wuhuizuo** [comment]
> /cc @flowbehappy @okJiang PTAL

sorry for requested wrongly.

**Review by flowbehappy — APPROVED** [review]

**Comment by wuhuizuo** [comment]
> @flowbehappy: adding LGTM is restricted to approvers and reviewers in OWNERS files.
> 
> In response to [this](https://github.com/pingcap/tiflow/pull/10820#pullrequestreview-1947852720):
> 
> > 
> 
> Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md). If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.

/hold

wait for [flowbehappy](https://github.com/flowbehappy) be a repo approver.

**Review by sdojjy — APPROVED** [review]

**Review by CharlesCheung96 — APPROVED** [review]

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Sort the member alphabetically.
+aliases:
+  sig-approvers-cdc-api:
+    - flowbehappy
```
Comment (flowbehappy) [inline OWNERS_ALIASES:4]:
```suggestion
    - hongyunyan
```

Comment (wuhuizuo) [reply OWNERS_ALIASES:4]: /cc @hongyunyan 
PTAL

**Review by hongyunyan — APPROVED** [review]
LGTM

**Pull request merged**